src/components/ConfirmModal.js

```diff
@@ -62,7 +66,8 @@ const ConfirmModal = props => (
             </Text>
 
             <Button
-                success
+                success={!props.danger}
```

Comment:
You are right, it will be better not inverted. The styles of `danger` version are overwriting the success so now we can use it with just `success` but as you mentioned before, we should refactor the `Button` component as a follow up.